Packages.props

```diff
@@ -24,7 +24,7 @@
 		<PackageReference Update="Microsoft.Azure.Kusto.Data" Version="9.0.4" />
 		<PackageReference Update="Microsoft.Azure.Kusto.Language" Version="9.0.4"/>
 		<PackageReference Update="Microsoft.SqlServer.Assessment"  Version="[1.0.305]" />
-		<PackageReference Update="Microsoft.SqlServer.Migration.Assessment"  Version="1.0.20210902.7" />
+		<PackageReference Update="Microsoft.SqlServer.Migration.Assessment"  Version="1.0.20211113.1085" />	
```

Comment:
Ignore this, this is a private build to test some stuff. Although at some point we actually do have to update this package